src/main/webapp/dev/js/common/const.js

```diff
@@ -18,7 +18,11 @@ const Const = {
 
 // Shared constants between frontend and backend
 
-// Subset of Bootstrap contextual colors for use in status messages and components of modals
+/**
+ * Subset of Bootstrap contextual colors for use in status messages and components of modals.
+ * @enum {BootstrapContextualColors}
+ */
+
```

Comment:
Remove this extra new line?

Reply:
Did you miss this?

Reply:
Really sorry, I missed this, will do it right away.